surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/LoaderVersionExtractor.java

```diff
@@ -42,6 +47,14 @@ public static String getJunitVersion() {
         return getVersionFromClassLoader(LIBRARY_JUNIT, currentThread().getContextClassLoader());
     }
 
+    public static String getJUnit5Version() {
+        return getVersionFromClassLoader(LIBRARY_JUNIT_5, currentThread().getContextClassLoader());
+    }
+
+    public static String getSurefireJUnit5Version() {
+        return getVersionFromClassLoader(LIBRARY_SUREFIRE_JUNIT_5, currentThread().getContextClassLoader());
+    }
```

Comment:
Do we really need these methods? To simulate the Surefire's behavior correctly, if the JUnit 5 provider is present, then we should delegate to this provider, no matter what is on the classpath...

Reply:
In fact needs to be removed.